### PR TITLE
dan.chiniara/Add dotted line to monitor status doc

### PR DESCRIPTION
**Pull request by djchinia**
### What does this PR do? What is the motivation?
Adds more context for the [dotted line](https://github.com/DataDog/documentation/assets/6266179/591b0d1d-c742-4480-88b0-641e5ee1a5d1) in the status bar for the Monitor Status page

dotted line
### Merge instructions

I'm checking with Product and Documentation Team before merging. 

- [ ] Please merge after reviewing

### Additional notes

- https://datadoghq.atlassian.net/browse/DOCS-6589
- https://docs-staging.datadoghq.com/dan.chiniara/Add-dotted-line-to-monitor-status-doc/monitors/manage/status

### CODE REVIEW

File: content/en/monitors/manage/status.md [hunk 1]
```diff
@@ -118,6 +118,15 @@ For further investigation into your metrics evolution, click **Open in a noteboo
 
 The notebook matches the monitor evaluation period time range and includes related logs where relevant.
 
+#### Follow monitor group retention
+
+Datadog keeps monitor groups available in the UI for 24 hours unless the query is changed. Host monitors and service checks that notify on on No Data are available for 48 hours. A dotted line appears for two reasons:
```
Comment (estherk15) [inline content/en/monitors/manage/status.md:123]:
```suggestion
Datadog keeps monitor groups available in the UI for 24 hours unless the query is changed. Host monitors and service checks that are configured to notify on missing data are available for 48 hours. A dotted line appears for two reasons:
```

File: content/en/monitors/manage/status.md [hunk 2]
```diff
@@ -118,6 +118,15 @@ For further investigation into your metrics evolution, click **Open in a noteboo
...
+
+- The new group is evaluated some time after the monitor is created. The dotted line is shown from the start of the time period to when the group was first evaluated.
```
Comment (estherk15) [inline content/en/monitors/manage/status.md:125]:
```suggestion
- The new group is evaluated some time after the monitor is created. The evaluation graph shows the dotted line from the start of the time period to when the group is first evaluated.
```

File: content/en/monitors/manage/status.md [hunk 3]
```diff
@@ -118,6 +118,15 @@ For further investigation into your metrics evolution, click **Open in a noteboo
...
+- The group stops reporting, drops out, and then starts reporting again. The dotted line appears from when the group dropped out to when it starts evaluating again.
```
Comment (estherk15) [inline content/en/monitors/manage/status.md:126]:
```suggestion
- The group stops reporting, drops out, and then starts reporting again. The dotted line appears from the time the group dropped out to when the group starts evaluating again.
```

**Review by estherk15 — APPROVED** [review]
Let me know if my suggestions alter the message of the docs!

**Comment by djchinia** [comment]
Thanks @estherk15! Added in your changes and changed the image to show more dotted lines/a highlighted group with a dotted line.

**Pull request merged**